package.json

```diff
@@ -72,10 +72,12 @@
     "ember-intl-relativeformat": "^2.4.0",
     "exists-sync": "0.0.3",
     "extend": "^3.0.0",
+    "has-unicode": "^2.0.1",
```

Comment:
I used yarn to install so lockfile update is apparently not necessary as some subdependency depends on it already. Likely npm via ember-cli...